app/src/main/kotlin/com/simplemobiletools/keyboard/helpers/Constants.kt

```diff
@@ -35,24 +38,30 @@ enum class ShiftState {
             }
         }
 
-        fun getCapitalizationOnDelete(context: Context, text: CharSequence?): ShiftState {
-            if (!context.config.enableSentencesCapitalization) {
-                return OFF
+        fun shouldCapitalize(context: Context, text: String?): Boolean {
+            //To capitalize first letter in textField
```

Comment:
what does this mean? Also if you want to write a comment, add a space after "//" and start with lowercase